frontend/src/App.js

```diff
@@ -275,22 +275,22 @@ const CourseText = (
   <>
     <h1 dangerouslySetInnerHTML={{__html: page.title}}/>
     {page.steps.slice(0, step_index + 1).map((part, index) =>
-      <div key={index} id={`step-text-${index}`}>
+      <div key={index} id={`step-text-${index}`} className={index > 0 ? 'pt-3' : ''}>
         <Markdown html={part.text} copyFunc={text => bookSetState("editorContent", text)}/>
-        <hr/>
+        <hr style={{ margin: '0' }}/>
       </div>
     )}
     <Messages {...{messages}}/>
 
-    <div>
+    <div className='pt-3'>
       {page.index > 0 &&
       <button className="btn btn-primary btn-sm previous-button"
               onClick={() => movePage(-1)}>
         Previous
       </button>}
       {" "}
       {page.index < Object.keys(pages).length - 1 && step_index === page.steps.length - 1 &&
-      <button className="btn btn-success next-button"
+      <button className="btn btn-success btn-sm next-button"
```

Comment:
I had the next button intentionally bigger than the previous button because it's probably needed more often. Do you think it looks too weird like that? If it needs to be consistent I think I'd prefer if they were both the default size, not small.

Reply:
I did notice the 'Next' button was bigger, but I didn't think it was on purpose. It does look kind of 'out of place' as it messes with the flow of the page. Both buttons being big or small would be consistent and would look better imo.

Reply:
OK, please remove the btn-sm from both then.

Reply:
Done, both buttons are now the same height.